### DISCUSSION TITLE
Use Drupal Pod on own Github Repo

### DISCUSSION THREAD

**Discussion by kitsunemax** [shaal/DrupalPod#63] [Q&A]
We use github for a drupal project, and it is not listed on the drupal.org website (or gitlab).

Can we use drupalpod to work on our github project repo, or it HAS to be via the drupal.org listings?

Thx, Max

**Comment by shaal** [comment]
Hi @MaxLynam

Drupalpod is a combination of a few technologies (Gitpod + ddev + bash scripts + browser extension), that were created with the goal of simplifying what it takes to start a Drupal contribution to 1 click.

Drupalpod could probably be enhanced with additional features, which I'm happy to learn about and discuss.

I'm trying to understand your question better, are you trying to use similar technology to develop a client's Drupal website? If yes, the setup can be much simpler than DrupalPod, you can look at an example here - https://github.com/shaal/fldc22/tree/gitpod

If you are trying to achieve something else, I'll be happy to help figure it out.

In any case, feel free to ask here, or you can also find me (@shaal) on #drupalpod channel in Drupal's Slack.

**Reply by kitsunemax** [reply]
Thanks for the reply @shaal   :)

It is about being able to use repositories on github ... for the development of projects based on Drupal core (& drupal projects/modules, etc).    It is Drupal, but we need to store the repository we are working on in Github, and not 

In this context, I guess it could be looked at as a client site, except it is our #techforgood projects, as opposed to something we sell to a client.  We are a non-profit and social enterprise, so many contributors are transient volunteers, so having it super-easy for them to contribute to drupal projects via service like drupalpod/gitpod will help manage transient volunteers, or contributors with limited bandwidth, to easily hop onto code and contribute - even on very small tasks.

I'll checkout the example you have linked to - and have joined the Drupal slack and channel - thank you again  :) 

Max

**Reply by shaal** [reply]
Feel free to ask any questions.
Thanks to ddev latest development, it became so easy to have a drupal project in Gitpod.
